scripts/pass_rate.py

```diff
@@ -76,9 +76,25 @@ def parse_report(report_path: pathlib.Path) -> ReportStats:
         except FileNotFoundError:
             pass
         stats.fixme += len(testsuite_fixme_tests)
-    deselected = get_deselected(report_path)
-    stats.skipped += deselected
-    stats.total += deselected
+
+    test_unskip = os.getenv('TEST_UNSKIP')
+    if test_unskip is None or test_unskip not in ("true", "false"):
```

Comment:
"Be consistent with your choice of string quote character within a file.", https://google.github.io/styleguide/pyguide.html#310-strings.
```suggestion
    if test_unskip is None or test_unskip not in ('true', 'false'):
```

Reply:
Done, thanks